content/en/docs/userguide/try_waypoint.md

```diff
@@ -64,32 +64,58 @@ sleep-9454cc476-86vgb            1/1     Running   0          62s
 
 6. Deploy waypoint:
 
-Deploy a waypoint for service account `bookinfo-reviews`, so any traffic to service `reviews` will be mediated by that waypoint proxy
+Deploy a waypoint for service `reviews`, so any traffic to that service will be mediated by that waypoint proxy
 
 ```bash
-[root@ ~]# istioctl x waypoint apply --service-account bookinfo-reviews
+[root@ ~]# istioctl experimental waypoint apply -n default --name reviews-svc-waypoint
```

Comment:
why change x ?

Reply:
Actually `x` is alias of `experimental` , ref: https://github.com/istio/istio/blob/master/istioctl/cmd/root.go#L149

I'll change back to `x` since it's more common.